http-client/README.md

```diff
@@ -39,3 +39,18 @@ Using aliases is similar to using filters:
         .withAlias(QuxClient.class);
 
 Note that the filters and aliases are unique to the binding, allowing you to have a different set for each one.
+
+The HTTP client also supports logging, which can be enabled with the `http-client.log.enabled` config property.
+Here is a sample log line:
+
+```
+2018-02-05T10:52:07.675-08:00   http    POST    http://[2401:db00:3020:b19a:face:0:27:0]:7777/v1/task/20180205_185203_00028_53xzu.3.66?summarize        200     1493    6437
+```
+A request log line has seven columns delimited with the tab character:
+- Request timestamp in the ISO 8601 format.
+- The protocol (http, https).
```

Comment:
Maybe we should have the version (http 1.1/2.0) instead of the `protocol` column.